### ISSUE TITLE
[0.8.0] Make temporal-system namespace visible through config

### ISSUE THREAD

**Issue by tsurdilo** [temporalio/ui#314]
<img width="372" alt="Screen Shot 2022-03-14 at 10 07 38 PM" src="https://user-images.githubusercontent.com/119422/158291717-9f011a98-683c-4f04-899a-3e77f9a790f1.png">

Would it be possible to structure this to temporal-system namespace is either not shown or is enabled via some sort of env var setting (to be shown). If is always shown it would be nice to always show the user defined namespaces first.

**Comment by tsurdilo** [comment]
Reopening because cannot confirm with 0.5.0, see https://github.com/temporalio/ui/issues/358

**Comment by feedmeapples** [comment]
should be fixed in v0.9.0

**Comment by feedmeapples** [comment]
to enable system-namespace run

```
docker run --network host -e TEMPORAL_SHOW_TEMPORAL_SYSTEM_NAMESPACE=true temporalio/ui:0.9.0
```
